plugins/event_notification/providers/http/lib/model/kHttpNotificationObjectData.php

```diff
@@ -37,15 +37,28 @@ class kHttpNotificationObjectData extends kHttpNotificationData
 	 */
 	protected $coreObject;
 
+	/**
+	 * prefix for sent string
+	 * @var string
+	 */
+	public $dataStringPrefix;
```

Comment:
@shpiNi  the properties $dataStringPrefix, $dataStringPostfix should be protected, just like all the other properties in the class.